deployments/liqo/templates/pre-delete-rbac.yaml

```diff
@@ -0,0 +1,55 @@
+{{- $predelete := (merge (dict "name" "pre-delete" "module" "pre-delete") .) -}}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
```

Comment:
I think those should also have the hook annotations, to be created only at deletion time